src/restartscript.cpp

```diff
@@ -528,7 +530,8 @@ void writeScript(const string& ckptDir,
                   "        . $DMTCP_SRUN_HELPER_SYNCFILE\n"
                   "        pass_slurm_helper_contact \"$DMTCP_LAUNCH_CKPTS\"\n"
                   "        rm $DMTCP_SRUN_HELPER_SYNCFILE\n"
-                  "        dmtcp_restart --join --coord-host $DMTCP_COORD_HOST"
+                  "        dmtcp_restart --join-coordinator"
+                              " --coord-host $DMTCP_COORD_HOST"
```

Comment:
In this case, it doesn't make sense to change the alignment.  The issue is that the second new line is a continuation of the first new line.  So, the only (bad) options are:

1.  Shift the second line all the way to the leftto line up.  The " --coord-host .." will stick out in the left margin.  Or,

2.  Shift the second line all the way to the left to line up, and add extra spaces to the "--coord-host" appears underneath "dmtcp_restart".  But in this case, the C++ source code looks good.  But the script that is generated will have lots of spaces in the middle of that line in the script.